server/src/test/java/io/unitycatalog/server/sdk/tempcredential/SdkTemporaryCredentialOperations.java

```diff
@@ -0,0 +1,39 @@
+package io.unitycatalog.server.sdk.tempcredential;
+
+import io.unitycatalog.client.ApiClient;
+import io.unitycatalog.client.ApiException;
+import io.unitycatalog.client.api.TemporaryCredentialsApi;
+import io.unitycatalog.client.model.*;
+
+public class SdkTemporaryCredentialOperations {
+  private final TemporaryCredentialsApi temporaryCredentialsApi;
+
+  public SdkTemporaryCredentialOperations(ApiClient apiClient) {
+    this.temporaryCredentialsApi = new TemporaryCredentialsApi(apiClient);
+  }
+
+  public TemporaryCredentials generateTemporaryModelVersionCredentials(
```

Comment:
i dont understand why this base class for cred operations is needed. 

generateTemporaryModelVersionCredentials is only going to be used only by one class SdkTemporaryModelVersionCredentialTest. Then why not define this method right in that class? 

Same for all table, volume, path?